webpack.config.js

```diff
@@ -26,7 +26,8 @@ module.exports = {
   },
   plugins: [
     new webpack.DefinePlugin({
-      SDK_VERSION: JSON.stringify(require('./package.json').version)
+      SDK_VERSION: JSON.stringify(require('./package.json').version),
+      STATE_TOKEN_COOKIE_NAME: JSON.stringify('oktaStateToken')
```

Comment:
For config stuff, I've been adding them to package.json i.e. https://github.com/okta/npm/blob/master/install-with-shrinkwrap/package.json#L43